recipe-server/webpack.config.js

```diff
@@ -10,11 +10,14 @@ var childProcess = require('child_process');
 const BOLD = '\u001b[1m';
 const END_BOLD = '\u001b[39m\u001b[22m';
 const production = process.env.NODE_ENV === 'production';
+const cssNamePattern = production ? '[name].[hash].css' : '[name].css';
```

Comment:
This is great. Someday I'd like to clean up this file and have something similar to django-configurations for specifying environmental settings for our webpack config.